java/ql/src/experimental/Security/CWE/CWE-470/LoadClassNoSignatureCheck.qhelp

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+If a vulnerable loads classes or code of any app based solely on the package name of the app without 
+first checking the package signature of the app, this could malicious app with the same package name 
+to be loaded through "package namespace squatting".
+If the victim user install such malicious app in the same device as the vulnerable app, the vulnerable app would load
+classes or code from the malicious app, potentially leading to arbitrary code execution.
```

Comment:
```suggestion
If an application loads classes or code from another app based solely on its package name without 
first checking its package signature, this could allow a malicious app with the same package name 
to be loaded through "package namespace squatting".
If the victim user install such malicious app in the same device as the vulnerable app, the vulnerable app would load
classes or code from the malicious app, potentially leading to arbitrary code execution.
```